app/controllers/frontend_log_controller.rb

```diff
@@ -33,6 +33,8 @@ class FrontendLogController < ApplicationController
     'IdV: personal key acknowledgment toggled' => :idv_personal_key_acknowledgment_toggled,
     'IdV: prepare submitted' => :idv_in_person_prepare_submitted,
     'IdV: prepare visited' => :idv_in_person_prepare_visited,
+    'IdV: selfie image added' => :idv_selfie_image_added,
+    'IdV: selfie image clicked' => :idv_selfie_image_clicked,
```

Comment:
With this one you're inadvertently starting [LG-12178](https://cm-jira.usa.gov/browse/LG-12178) 😁

Reply:
Mmmm, good point. But I want this now - it's causing confusing errors in the console 😆

Reply:
Yeah I think if it's causing issues in your current work - it makes sense to fix now! The ticket has some info on some properties we want on that event as well if it's easy. Otherwise if it's hard, we can save adding those properties for when the ticket is prioritized.

It's also a little bit different order than they're laid out, so we might have to update https://cm-jira.usa.gov/browse/LG-11982 to include this event once this is merged.